src/components/reusable/table/table-row.scss

```diff
@@ -4,6 +4,9 @@
   display: table-row;
   vertical-align: middle;
 }
+:host(:hover) {
+  background-color: var(--kd-color-background-ui-soft) !important;
```

Comment:
Can it be done without `!important`? 
I think it might be nice to also add a little `transition` like `background-color 150ms ease-out`.

Reply:
Nope I tried but so many css overrides inside table. As table has so many :hover properties overriden like active (hover) disabled etc.
sure we can add transition.

Reply:
adding 200ms. I think for table it's 200ms by peter in rest of the places.